mlos_bench/mlos_bench/tests/optimizers/toy_optimization_loop_test.py

```diff
@@ -73,7 +73,7 @@ def test_mock_optimization_loop(mock_env_no_noise: MockEnv, mock_opt: MockOptimi
         "vmSize": "Standard_B2ms",
         "idle": "halt",
         "kernel_sched_migration_cost_ns": 117026,
-        "kernel_sched_latency_ns": 149827700,
+        "kernel_sched_latency_ns": 149827706,
```

Comment:
These really should be getting returned on the quantization boundary, no?

Reply:
The proper quantization boundary here should be something like `100000000`..

The problem is that neither Mock Optimizer nor FLAML respect quantization (or many other ConfigSpace/Tunable parameters like distributions). Mock just samples uniformly from the Tunables' range, and for FLAML we have a very limited conversion from ConfigSpace to FLAML hyperparameters (and I don't know if they have any support for quantiztion etc.).

I still have some faint hope for SMAC3, but I cannot figure out why it does not call `ConfigSpace.sample_configuration()` like I think it should - maybe it also looks directly into the hyperparameters' ranges etc. and uses ConfigSpace only to validate the proposed configuration (and we don't monkey patch that part to enforce quantization somehow)